.pre-commit-config.yaml.jinja

```diff
@@ -1,6 +1,16 @@
 {# We cannot use the same pre-commit config as MQT's because paths don't match
    and it's designed only for Odoo 13.0+, so we design our own version, which
    should be as close as possible to MQT's -#}
+
+{#- This namespace holds repo versions #}
+{%- set proj_rev = namespace() %}
+
+{%- if odoo_version < 16 %}
+  {%- set proj_rev.pylint_odoo = "v8.0.0" %}
```

Comment:
Why is this pinned to version 8.0.0?
Scrolling trough changes notes of pylint_odoo I did'nt see any breaking changes.

I'm succesfully using version 8.0.20 for odoo V14

Reply:
The available checks was changed... if you run pylint-odoo in verbose mode you will see warnings messages.

Reply:
Let's keep it for now in that version. You can make a PR updating to that version and changing the checks names.